package.json

```diff
@@ -5,16 +5,22 @@
   "main": "index.js",
   "scripts": {
     "coverage": "npm run db-migrate && mocha --reporter mochawesome --timeout=20000 --require babel-register --exit",
-    "db-migrate": "sequelize db:migrate:undo:all && sequelize db:migrate",
+    "db-migrate": "node_modules/.bin/sequelize db:migrate:undo:all && sequelize db:migrate",
     "lint": "eslint routes models config test index.js",
     "test": "npm run db-migrate && mocha test/index.js --timeout=20000 --require babel-register --exit",
-    "start": "sequelize db:migrate && babel-node index.js",
+    "start": "node_modules/.bin/sequelize db:migrate && babel-node index.js",
```

Comment:
babel-node should not be used in production. The code should be "transpiled" and run by Node